scout/parse/variant/callers.py

```diff
@@ -13,6 +13,14 @@ def parse_callers(variant, category="snv"):
     2. If a svdb_origin tag (pipe separated) is found, callers listed will be marked Pass
     3. If a set tag (dash separated, GATK CombineVariants) is found, callers will be marked Pass or Filtered accordingly
 
+    If the FILTER status is not PASS (e.g. None - cyvcf2 FILTER is None if VCF file column FILTER is "PASS"),
```

Comment:
I dunno, this function is getting long. If I didn't have the thought about a new parsing module nudging I might split/refactor it. Say if you feel it is way long already.